### ISSUE TITLE
[a11y] delete action in settings does not have a proper label

### ISSUE THREAD

**Issue by susnux** [nextcloud/groupfolders#4462]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. open settings
2. check accessibility information on the trashbin action

### Expected behaviour
Have either `aria-label` or visual text inside the link element.

### Actual behaviour
Only "title" attribute is provided which is often ignored by screen readers

<img width="658" height="481" alt="Image" src="https://github.com/user-attachments/assets/8f632b58-d35e-4557-a8c5-5c071cb88ae7" />

**Comment by kristian-zendato** [comment]
@susnux , @ericfischereu This issue is not reproducible. Looks liked already fixed.

**Comment by susnux** [comment]
Yes seems to be fixed since https://github.com/nextcloud/groupfolders/pull/4989
